website_sale_medical_medicament_us/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Medicament Sales - Brand/Generic Links',
+    'summary': 'Links to brand/generic alternatives on medicament shop pages',
+    'version': '9.0.1.0.0',
+    'category': 'Medical',
+    'website': 'https://laslabs.com/',
+    'author': 'LasLabs, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
```

Comment:
Add `'auto_install': True`. This forces install as long as all dependencies are installed, which is desired for the US modules